package.json

```diff
@@ -1,34 +1,31 @@
 {
   "name": "golden-sample-app",
   "version": "0.0.0",
+  "license": "MIT",
   "scripts": {
-    "ng": "ng",
-    "start": "ng serve",
-    "build": "ng build",
-    "build:stats": "ng build --configuration production --stats-json",
-    "analyze": "webpack-bundle-analyzer dist/golden-sample-app/stats.json",
+    "ng": "nx",
+    "postinstall": "node ./decorate-angular-cli.js && ngcc --properties es2015 browser module main",
+    "start": "nx serve",
+    "build": "nx build",
+    "test": "nx test --code-coverage",
     "build:docker": "docker build -t golden-sample-app:latest .",
     "start:docker": "docker run -p 8080:8080 golden-sample-app",
-    "xi18n": "ng extract-i18n golden-sample-app --output-path=src/locale && xliffmerge --profile src/xliffmerge.json nl-NL",
-    "test": "ng test --code-coverage=true",
-    "lint": "ng lint",
-    "e2e": "ng e2e",
-    "build:journeys": "ng build transactions-journey && ng build transfer-journey && ng build ach-positive-pay-journey",
-    "test:transfer:watch": "ng test transfer-journey --watch",
-    "test:transactions:watch": "ng test transactions-journey --watch"
+    "xi18n": "ng extract-i18n golden-sample-app --output-path=apps/golden-sample-app/src/locale && xliffmerge --profile apps/golden-sample-app/src/xliffmerge.json nl-NL",
+    "e2e": "npx playwright test",
+    "lint": "nx run-many --all --target=lint"
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~12.2.14",
-    "@angular/cdk": "^12.2.13",
-    "@angular/common": "~12.2.14",
-    "@angular/compiler": "~12.2.14",
-    "@angular/core": "~12.2.14",
-    "@angular/forms": "~12.2.14",
-    "@angular/localize": "~12.2.14",
-    "@angular/platform-browser": "~12.2.14",
-    "@angular/platform-browser-dynamic": "~12.2.14",
-    "@angular/router": "~12.2.14",
+    "@angular/animations": "~13.1.1",
```

Comment:
why did you update to angular 13? if this is a sample for all the developers, we should keep the state of this the same as developers are doing, so that they can base this as their source of truth. 

Maybe we can keep an unstable branch or something with the latest angular for research and testing.

Reply:
Daniel did it to test if there was an issue with jest for testing. If it's working ok I don't think we need to necessarily revert it, we want to showcase "future state" as well

Reply:
I am fine to do this testing for "future state" but not for something that is a reference to other developers. Here we show how to handle journeys, authentication, entitlements, deploying, communication, data handling, core libraries implementations, building configurations, best practices and now monorepo setup. All of this is based on angular, if you use a future version, that would also force us to upgrade the peer dependencies. And therefore this will no longer reflect what we want developers to do in the capabilities. But fortunately we can make a branch for future state and experiment there. This is something we all agree in web-foundation during our last meeting.

Reply:
In Master branch, @santam85, I would like to keep the current version but anyways this doesn't stop us of having an experimental branch on which we can start working updating Angular as well as any other things we would like to add to master sooner or later. We do link the code inside our documentation in community and if we do changes like this one it might end with confusing code versus documentation.

Reply:
Sounds like the sensible way to do it.